client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescriptions.tsx

```diff
@@ -123,10 +125,21 @@ const getPrescriptionKey = (prescriptions: Prescription[], openedPrescriptionId:
   return get(prescription, 'id');
 };
 
+const getAnalysisTraduction = (display: string, concepts: Concept[], lang:string ) => {
```

Comment:
change function name: ex) translateAnalysis. It is a personal taste, but I try not to use the get prefix all the time.

You already used that function in an other component. It is probably time to extract it to avoid code repeat.